### DISCUSSION TITLE
Contribution Policy for Config Metadata?

### DISCUSSION THREAD

**Discussion by The00Dustin** [zwave-js/zwave-js#7275] [General] [answered]
The config file https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x026e/vcz1.json is missing inclusion, exclusion, and wakeup.  I'd be happy to add these and submit a PR, but I'm not sure about policy with regard to matching documentation.  My specific questions follow:

The manual delivered with my device doesn't have exclusion (which may be the same as inclusion anyway), and the inclusion differs slightly from the manual linked in the config db browser.  Should I favor  one verbiage over the other?

Neither manual includes the wake instructions, but I have personally confirmed how to wake the device.  Is it OK to write the instructions myself, or is it better to leave them missing?

**Comment by AlCalzone** [comment, marked as answer]
If you have instructions that work for you, just go ahead and add them. Better than missing them or having wrong instructions.
